docs/content/preview/manage/upgrade-deployment.md

```diff
@@ -18,9 +18,27 @@ If you plan to script this in a loop, then a pause of approximately 60 seconds i
 
 Your `data/log/conf` directories are generally stored in a separate location which stays the same across the upgrade so that the cluster data, its configuration settings are retained across the upgrade.
 
+## Before you begin
+
+Before starting the upgrade, review the following major changes to YugabyteDB. Depending on the upgrade you are planning, you may need to make changes to your automation.
+
+### Upgrading to v2.16.0
```

Comment:
The titles "Upgrading to v2.16.0" and "Upgrading to v2.18.0" seems confusing considering that we are on a preview page.